akka-remote/src/main/scala/akka/remote/serialization/MiscMessageSerializer.scala

```diff
@@ -12,9 +12,13 @@ class MiscMessageSerializer(val system: ExtendedActorSystem) extends SerializerW
 
   private lazy val serialization = SerializationExtension(system)
 
+  private val NoneSerialized = ContainerFormats.Some.newBuilder().build().toByteArray
```

Comment:
I guess this is the same as `Array.empty[Byte]`, perhaps we should use empty array for some more explicitly